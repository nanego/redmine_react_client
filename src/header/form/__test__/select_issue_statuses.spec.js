import React from 'react';
import SelectIssueStatuses from '../select_issue_statuses';
import renderer from 'react-test-renderer';

var updateSelectedFilters = function(){};

it('should render SelectIssueStatuses', () => {
  const component = renderer.create(
      <SelectIssueStatuses selected_filters={{}} updateSelectedFilters={updateSelectedFilters} />
  );
  let select = component.toJSON();
  expect(select).toMatchSnapshot();
});
