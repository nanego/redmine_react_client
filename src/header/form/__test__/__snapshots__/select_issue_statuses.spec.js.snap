exports[`test should render SelectIssueStatuses 1`] = `
<div
  aria-busy={false}
  aria-disabled={undefined}
  aria-expanded={false}
  className="ui fluid search selection dropdown"
  onBlur={[Function]}
  onChange={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  role="combobox"
  tabIndex={undefined}>
  <select
    aria-hidden="true"
    multiple={undefined}
    name={undefined}
    type="hidden"
    value="">
    <option
      value="" />
    <option
      value={1}>
      New
    </option>
    <option
      value={2}>
      In Progress
    </option>
    <option
      value={3}>
      Done
    </option>
    <option
      value={4}>
      Feedback
    </option>
    <option
      value={5}>
      Closed
    </option>
    <option
      value={6}>
      Rejected
    </option>
    <option
      value={undefined} />
  </select>
  <input
    aria-autocomplete="list"
    autoComplete="off"
    className="search"
    name="-search"
    onChange={[Function]}
    style={
      Object {
        "width": undefined,
      }
    }
    tabIndex={0}
    type="text"
    value="" />
  <div
    className="default text">
    Issue Statuses
  </div>
  <i
    aria-hidden="true"
    className="dropdown icon" />
  <div
    aria-multiselectable={undefined}
    className="menu transition"
    role="listbox">
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={true}
      className="selected item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        New
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        In Progress
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        Done
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        Feedback
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        Closed
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      }>
      <span
        className="text">
        Rejected
      </span>
    </div>
    <div
      aria-checked={false}
      aria-disabled={undefined}
      aria-selected={false}
      className="item"
      onClick={[Function]}
      role="option"
      style={
        Object {
          "pointerEvents": "all",
        }
      } />
  </div>
</div>
`;
